pages/imagePreview.css

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
I'm sure this license text got copied from some example code, but, can it be removed?  In particular the "License.txt" file it mentions isn't part of this project.

Reply:
It wasn't example code its the same CSS that is used in vscode itself with two minor modifications to be useful in our context.  So I don't know what rules apply.

Reply:
OK.  We'll leave it alone for now.